packages/components/src/DateTimePickers/pickers/DateTimePicker/DateTimePicker.component.js

```diff
@@ -189,6 +189,7 @@ class DateTimePicker extends React.Component {
 				{viewElement}
 				<div className={theme.footer}>
 					<Action
+						id="date-picker-today-btn"
```

Comment:
this mean I can t have two datetimepicker in the same page ☝️ .
please use className instaed

Reply:
moved.